README.md

```diff
@@ -1,11 +1,11 @@
 # External Gitpod Configuration
 
 [Gitpod](https://gitpod.io) is an online IDE for GitHub.
-This repository contains externally maintained `.gitpod` files for projects on GitHub.
+This repository contains externally maintained `.gitpod.yml` files for projects on GitHub.

```

Comment:
@jankeromnes Are we gonna have separate definitely-gp repos for other platforms like `GitLab` and `Bitbucket`?